app/browser/api/ledger.js

```diff
@@ -2072,6 +2054,22 @@ const getContributionAmount = (state) => {
 }
 
 const onInitRead = (state, parsedData) => {
+  const isBTC = parsedData &&
+    parsedData.properties &&
+    parsedData.properties.wallet &&
+    parsedData.properties.wallet.keychains
+
+  if (isBTC) {
```

Comment:
I think we should add verbose logging here- something like:
`Bitcoin wallet detected; backing up to "ledger-state-btc.json" and creating new BAT wallet`